pipeline.py

```diff
@@ -159,14 +171,53 @@ def pipeline(
         output_data_task = pvc_to_model_op(
             pvc_path="/output/model",
             )
+
+        # MMLU Evaluation of models
+
+        models_list_task = list_models_in_directory_op(
+            models_folder="/output/model/model/hf_format",
+        )
+
+        models_list_task.after(kubectl_wait_task)
+
+        mount_pvc(
+            task=models_list_task, pvc_name=output_pvc_task.output, mount_path="/output/model"
+        )
+
+        run_mmlu_task = run_mmlu_op(
+            models_list=models_list_task.output,
+            models_path_prefix = "/output/model/model/hf_format", 
+            mmlu_tasks_list=mmlu_tasks_list,
+            model_dtype=model_dtype,
+            few_shots=few_shots,
+            batch_size=batch_size,
+            device=device,
+        )
+
+        mount_pvc(
+            task=run_mmlu_task, pvc_name=output_pvc_task.output, mount_path="/output/model"
+        )
+
+        load_mmlu_results_task = load_mmlu_results_op(
+            mmlu_output=run_mmlu_task.outputs['mmlu_output'],
+        )
+
+        best_model = run_mmlu_task.outputs['best_model']
+        best_score = run_mmlu_task.outputs['best_score']
+        print(f"Best Model: {best_model} with score: {best_score}")
+
+        run_mmlu_task.set_accelerator_type('nvidia.com/gpu')
+        run_mmlu_task.set_accelerator_limit(1)
+
         output_data_task.after(kubectl_wait_task)
         output_model_task.set_caching_options(False)
+        output_model_task.after(run_mmlu_task)
```

Comment:
does this .after() task conflict with line 166?
`models_list_task.after(kubectl_wait_task)`
Do we need 166 if we are wating until after run_mmlu_task to copy data from PVC to s3?

Reply:
`models_list_task` has to happen after `kubectl_wait_task` bc it lists the resulting models from training that are placed in the PVC. I removed the `output_model_task.after(run_mmlu_task)` because that isn't necessary to run mmlu - but we might not want to push all trained models to S3 if the only 1 that is important is the best-mmlu-score model. Instead, the workflow can obtain the best-scored model from MMLU _then_ only push the 1 winner. But that can be decided later - so removed that change here!